src/components/molecules/Settings/SettingPage/index.tsx

```diff
@@ -35,36 +40,104 @@ const SettingPage: React.FC<Props> = ({
         center={center}
       />
       <Wrapper3>
-        <Navigation team={currentTeam} project={currentProject} />
-        <Content>{children}</Content>
+        <LeftWrapper>
+          <Navigation team={currentTeam} project={currentProject} />
+        </LeftWrapper>
+        <RightWrapper>
+          <Wrapper5>
+            <DeviceMenu>
+              <MenuIcon
+                icon={isOpen ? "cancel" : "menuForDevice"}
+                size={32}
+                onClick={handleClick}
+              />
+              {isOpen && (
+                <Menu>
+                  <Navigation team={currentTeam} project={currentProject} />
+                </Menu>
+              )}
+            </DeviceMenu>
+            {children}
+          </Wrapper5>
+        </RightWrapper>
       </Wrapper3>
     </Wrapper>
   );
 };
 
+const StyledHeader = styled(Header)`
+  position: fixed;
+  box-shadow: 0 4px 24px #1d1d1d;
+  z-index: ${props => props.theme.zIndexes.settingHeader};
+`;
+
 const Wrapper = styled.div`
   height: 100%;
-  background-color: ${props => props.theme.colors.bg[2]};
-  overflow: scroll;
+  background-color: ${({ theme }) => theme.colors.bg[2]};
+  display: flex;
+  flex-direction: column;
 `;
 
 const Wrapper3 = styled.div`
-  display: flex;
-  flex: 1;
-  margin-top: 18px;
-  padding: 20px 60px 120px 60px;
+  height: 100%;
+  padding-top: 48px;
+  overflow: auto;
 `;
 
-const Content = styled.div`
-  flex: 1;
-  margin: 0 auto;
-  padding: 0 50px;
-  max-width: 943px;
+const LeftWrapper = styled.div`
+  width: 264px;
+  height: 100%;
+  background-color: ${({ theme }) => theme.colors.bg[3]};
+  position: fixed;
+
+  @media only screen and (max-width: 1024px) {
+    display: none;
+  }
+`;
+
+const RightWrapper = styled.div`
+  margin-left: 264px;
+  padding: 32px 64px;
   box-sizing: border-box;
+
+  @media only screen and (max-width: 1024px) {
+    margin: 0 auto;
+  }
+`;
+
+const Wrapper5 = styled.div`
```

Comment:
Any reason we have Wrapper, Wrapper3 and Wrapper5? Can you name them more clearly?

Reply:
yes, I these wrappers are for structures, I name them more clearly, any good ideas? @KaWaite